tune/tune.c

```diff
@@ -97,18 +97,18 @@ static int exfat_set_volume_label(struct exfat_blk_dev *bd,
 {
 	struct exfat_dentry vol;
 	int nbytes;
-	__u16 volume_label[11];
+	__u16 volume_label[VOLUME_LABEL_MAX_LEN];
 	int volume_label_len;
 
 	volume_label_len = exfat_utf16_enc(label_input,
-			volume_label, sizeof(volume_label));
+			volume_label, VOLUME_LABEL_MAX_LEN);
 	if (volume_label_len < 0) {
 		exfat_err("failed to encode volume label\n");
 		return -1;
 	}
 
 	vol.type = EXFAT_VOLUME;
-	memset(vol.vol_label, 0, 22);
+	memset(vol.vol_label, 0, VOLUME_LABEL_MAX_LEN * 2);
```

Comment:
`memset(vol.vol_label, 0, sizeof(vol.vol_label));` should be correct way how to erase whole `vol.vol_label` buffer